pkg/storage/disk/disk.go

```diff
@@ -0,0 +1,41 @@
+package disk
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/gomods/athens/pkg/storage"
+)
+
+// ListerSaver is the only interface defined by the disk storage. Use
+// NewListerSaver to create one of these. Everything is all in one
+// because it all has to share the same tree
+type ListerSaver interface {
```

Comment:
don't want to brag about naming so soon but can't this be just a Storage or StorageSystem? ListerSaver seems very weird to me